apps/klasses/models.py

```diff
@@ -35,7 +35,7 @@ class Klass(models.Model):
     instructor = models.ForeignKey('profiles.Instructor', related_name='klasses', on_delete=models.CASCADE)
 
     title = models.CharField(max_length=60, null=False, blank=False, default="New Course")
-    course_number = models.SlugField(max_length=60, null=False, blank=False, unique=True, default=None)
+    course_number = models.SlugField(max_length=60, null=False, blank=False, unique=False, default=None)
```

Comment:
can remove this kwarg if using default?

Reply:
Also `null=False` but you are setting `default=None`?

Reply:
Great point I'll remove the default setting.